examples/monaco-graphql-webpack/src/index.ts

```diff
@@ -78,9 +88,13 @@ const operationEditor = monaco.editor.create(
   document.getElementById('operation') as HTMLElement,
   {
     model,
+    automaticLayout: true,
   },
 );
 
+// @ts-ignore
+monaco.languages.graphql.graphqlDefaults.setSchemaUri(SCHEMA_URL);
```

Comment:
@cshaver if we do this inside schema context, then it will reload schema and validations, completion, etc everything!!!